<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotSettings</class>
 <widget class="QDialog" name="PlotSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="xAxisGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>333</width>
     <height>79</height>
    </rect>
   </property>
   <property name="title">
    <string>X Axis</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="xAxisLabel">
      <property name="text">
       <string>Label:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="xAxisRangeLabel">
      <property name="text">
       <string>Range:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="xAxisRangeLowEdit">
      <property name="placeholderText">
       <string>Low</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="xAxisRangeHighEdit">
      <property name="placeholderText">
       <string>High</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="xAxisLabelEdit">
      <property name="placeholderText">
       <string>X Axis Label</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="yAxisGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>333</width>
     <height>79</height>
    </rect>
   </property>
   <property name="title">
    <string>Y Axis</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="yAxisLabel">
      <property name="text">
       <string>Label:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="yAxisLabelEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Y Axis Label</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="yAxisRangeLabel">
      <property name="text">
       <string>Range:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="yAxisRangeLowEdit">
      <property name="placeholderText">
       <string>Low</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="yAxisRangeHighEdit">
      <property name="placeholderText">
       <string>High</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="saveSettingsPushButton">
      <property name="text">
       <string>Save Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelSettingsPushButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
